includes/Admin/Settings_Page.php

```diff
@@ -0,0 +1,270 @@
+						<div class="ai-experiments__card-heading">
+							<h2><?php esc_html_e( 'General Settings', 'ai' ); ?></h2>
+							<p class="description">
+								<?php esc_html_e( 'Control whether AI experiments are enabled for your site. When disabled, all experiments will be inactive regardless of their individual settings.', 'ai' ); ?>
+							</p>
+						</div>
+
+						<div class="ai-experiments__toggle">
+							<label class="components-toggle-control">
+								<input
+									type="checkbox"
+									name="<?php echo esc_attr( self::GLOBAL_OPTION ); ?>"
+									id="<?php echo esc_attr( self::GLOBAL_OPTION ); ?>"
+									value="1"
+									<?php checked( $global_enabled ); ?>
+								/>
+								<span class="ai-experiments__toggle-label">
+									<?php esc_html_e( 'Enable Experiments', 'ai' ); ?>
+								</span>
+							</label>
+						</div>
+					</div>
+
+					<!-- Individual Experiments Section -->
+					<?php if ( ! empty( $this->registry->get_all_experiments() ) ) : ?>
+						<div class="ai-experiments__card">
+							<div class="ai-experiments__card-heading">
+								<h2><?php esc_html_e( 'Available Experiments', 'ai' ); ?></h2>
+								<p class="description">
+									<?php esc_html_e( 'Try out the following experiments to see how AI can help your site.', 'ai' ); ?>
+								</p>
+
+								<?php if ( ! $global_enabled ) : ?>
+									<div class="notice notice-info inline ai-experiments__notice">
+										<p><?php esc_html_e( 'Enable experiments above to configure individual experiment settings.', 'ai' ); ?></p>
+									</div>
+								<?php endif; ?>
+							</div>
+
+							<ul class="ai-experiments__list">
+								<?php foreach ( $this->registry->get_all_experiments() as $experiment ) : ?>
+									<?php
+									$experiment_id      = $experiment->get_id();
+									$experiment_option  = "ai_experiment_{$experiment_id}_enabled";
+									$experiment_enabled = (bool) get_option( $experiment_option, false );
+									$disabled_class     = ! $global_enabled ? 'ai-experiments__item--disabled' : '';
+									?>
+									<li class="ai-experiments__item <?php echo esc_attr( $disabled_class ); ?>">
+										<div class="ai-experiments__item-header">
+											<label class="components-toggle-control">
+												<input
+													type="checkbox"
+													name="<?php echo esc_attr( $experiment_option ); ?>"
+													id="<?php echo esc_attr( $experiment_option ); ?>"
+													value="1"
+													<?php checked( $experiment_enabled ); ?>
+													<?php disabled( ! $global_enabled ); ?>
+												/>
+												<span>
+													<strong><?php echo esc_html( $experiment->get_label() ); ?></strong>
+												</span>
+											</label>
+										</div>
+										<?php if ( $experiment->get_description() ) : ?>
+											<p class="description">
+												<?php echo esc_html( $experiment->get_description() ); ?>
```

Comment:
I believe, we should support anchor links in the description just to ensure that we can link back to a documentation page where necessary for the end users. We can see anchor links in the description for existing WP admin settings as you can check under Settings > AI Credentials or Settings > Reading or any other admin page.

Here is an screenshot example of how the current settings is already supporting the anchor links along with the description text:
<img width="1648" height="242" alt="CleanShot 2025-11-17 at 19 55 38@2x" src="https://github.com/user-attachments/assets/05f77cb0-ed47-44a2-b12e-a08ee57d806e" />

Reply:
+1, let's use `wp_kses()` and allow a few common inline tags, at least anchors.